sdk/identity/identity-vscode/tsconfig.json

```diff
@@ -0,0 +1,14 @@
+{
+  "extends": "../../../tsconfig.package",
+  "compilerOptions": {
+    "target": "es6",
+    "lib": ["DOM"],
```

Comment:
similarly, this shouldn't work in the browser, yeah?

Reply:
For `vscode` it's fine. For `cache-persistence` it is required to build the tests, which reach into the identity source code during the build.